geps/gep-2907/index.md

```diff
@@ -30,29 +29,15 @@ in scope for this GEP. In the future, this GEP may be expanded to include:
 
 ## Introduction
 
-### Where TLS could be configured
-We have three different places where we might want to configure TLS:
-
-#### A. Gateways
-Config attached at this layer will apply to everything behind a single address
-such as a virtual IP.
+### TLS mode
 
-#### B. Gateway Listeners
-Each Gateway contains multiple “Listeners”. Each HTTPS Listener in a Gateway
-must have a unique combination of protocol, port, and SNI (Server Name
-Indication). TLS configuration attached at this layer should be [isolated via
-SNI](../../guides/implementers.md#listener-isolation).
+TLS can be configured with two distinct modes:
+* **Passthrough**: The gateway forwards encrypted traffic directly to the backend from the entity originating the connection (called later a client). The TLS connection is established between the client and the backend; the gateway does not participate in the TLS handshake.
 
-#### C. BackendTLSPolicy
-This policy allows us to attach unique TLS configuration per Backend. Depending
-on the organization, this policy may be owned by the application owner or the
-cluster operator. Note that this configuration will be used by all Gateways
-(potentially from different implementations) that are connecting to the backend.
+* **Terminate**: The gateway acts as a man-in-the-middle, terminating the client's TLS connection and originating a separate connection to the backend when routing the traffic. Both connections can independently support TLS and mutual TLS due to their distinct TLS configurations. The gateway's dual role as a client or server affects the placement of TLS settings needed for given connection.
```

Comment:
Nit: Kubernetes resource names should be capitalized throughout, so "Gateway" in this case.